components/publish/PublishWidget.vue

```diff
@@ -504,6 +504,21 @@ onDeactivated(() => {
 </template>
 
 <style scoped>
+  .publish-button {
+    background: #592ACB;
+    color: #fff;
+    border-radius: 360px;
+
+    &:hover {
+      background:#7542E5;
+      color: #fff;
+    }
+  }
+  .dark .publish-button {
+    background: #CB9EFF;
+    color: #000;
+  }
```

Comment:
If we use color variables for these, we shouldn't have to target the dark theme specifically

Reply:
I agree...just trying to bring over specific values from Figma per design request without changing other parts of the app. I can try to convince them to leave the current color values as is.